openedx/core/djangoapps/discussions/migrations/0002_auto_20201029_2143.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.16 on 2020-10-29 21:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('discussions', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='discussionproviderconfig',
+            name='provider',
+            field=models.CharField(choices=[('cs_comments', 'Inbuilt Discussion Forums')], db_index=True, help_text='The discussion tool/provider.', max_length=100, verbose_name='Discussion provider'),
```

Comment:
Note that we want to change the friendly name of the built-in discussions to "edX Discussions", as per @marcotuts's comment above.  (Appears twice in this file)

Reply:
Will do.

Reply:
Done.